README.md

```diff
@@ -33,43 +36,56 @@ Refinery is designed to sit within your infrastructure where all traces can reac
 
 Refinery processes must be able to communicate with each other to concentrate traces on single servers.
 
-Within your application (or other Honeycomb event sources) you would configure the `API Host` to be http(s)://load-balancer/. Everything else remains the same (api key, dataset name, etc. - all that lives with the originating client).
+Within your application (or other Honeycomb event sources), you would configure the `API Host` to be `http(s)://load-balancer/`. Everything else remains the same, such as API key, dataset name, and so on since all that lives with the originating client.
 
-### Minimum configuration
+### Minimum Configuration
 
-Every Refinery instance should have a minimum of 2GB RAM and at least 2 cores, preferably more. Refinery clusters usually run best with fewer, larger servers; for large installations, 16 cores and 16GB of RAM per server is not unusual (with appropriate tuning).
+Every Refinery instance should have a minimum of:
 
+- a `linux/amd64` or `linux/arm64` operating system
+- 2GB RAM for each server used
+- Access to 2 cores for each server used
+
+In many cases, Refinery only needs one node.
+Refinery clusters usually run best with fewer, larger servers; for large installations, 16 cores and 16GB of RAM per server is not unusual (with appropriate tuning).
+If experiencing a large volume of traffic, you may need to scale out to multiple nodes, and likely need a small Redis cluster to handle scaling.
```

Comment:
I am uncomfortable with this phrasing. If you have more than 1 node, you need redis (unless you're really hardcore but I don't want to encourage it). You probably don't need more than one redis, though, unless you're just paranoid.

Reply:
@kentquirk How about this then? It's the most recent version of the same content in the Honeycomb docs site [as seen here](https://docs.honeycomb.io/manage-data-volume/refinery/set-up/#system-requirements): 
> In many cases, Refinery only needs one node. If experiencing a large volume of traffic, you may need to scale out to multiple nodes, and likely need a small Redis instance to handle scaling.

Reply:
Refer to suggested change below.